.github/workflows/build_and_test.yml

```diff
@@ -8,11 +8,12 @@ on:
   push:
     branches:
       - develop_pwa
+  workflow_dispatch:
 
 env:
   CI: true
   MONGO_URL: mongodb://localhost:27017
-  TOOL_NODE_FLAGS: --max_old_space_size=4096
+  TOOL_NODE_FLAGS: --max_old_space_size=8192 --optimize_for_size --gc-interval=100
```

Comment:
Do we think this helped?  As I recall it did not originally help and that our issue just seemed to fix itself?

Reply:
Nope It didn't work